src/components/button/index.js

```diff
@@ -1,29 +1,37 @@
 import React, { Component } from 'react';
 import { translate } from 'react-i18next';
-import style from './style.styl';
-
 import Modal from '../modal'
+import style from './style.styl';
 
 class Button extends Component {
+
+    static defaultProps = {
+        type: 'normal',
+        icon: false,
+        loading: false,
+        apply: () => {}
+    }
```

Comment:
we should probably add `prop-types` https://facebook.github.io/react/docs/typechecking-with-proptypes.html